modules/apps/29-fee/keeper/msg_server.go

```diff
@@ -92,6 +92,10 @@ func (k Keeper) PayPacketFee(goCtx context.Context, msg *types.MsgPayPacketFee)
 		return nil, err
 	}
 
+	if err = k.bankKeeper.IsSendEnabledCoins(ctx, msg.Fee.Total()...); err != nil {
+		return nil, err
+	}
```

Comment:
nit: you could reinitialise the error since its used within the scope of the `if` but don't think its required

```suggestion
	if err := k.bankKeeper.IsSendEnabledCoins(ctx, msg.Fee.Total()...); err != nil {
		return nil, err
	}
```